build.sbt

```diff
@@ -2322,8 +2322,9 @@ lazy val `std-base` = project
     Compile / packageBin / artifactPath :=
       `base-polyglot-root` / "std-base.jar",
     libraryDependencies ++= Seq(
-      "org.graalvm.polyglot" % "polyglot"                % graalMavenPackagesVersion,
-      "org.netbeans.api"     % "org-openide-util-lookup" % netbeansApiVersion % "provided"
+      "org.graalvm.polyglot"      % "polyglot"                % graalMavenPackagesVersion,
+      "org.apache.httpcomponents" % "httpclient"              % httpComponentsVersion,
```

Comment:
Are we adding dependency on an external library to `std-base` just be able to emit HTTP request? What's wrong with [JDK only solution](https://docs.oracle.com/en/java/javase/11/docs/api/java.net.http/java/net/http/HttpClient.html)?

Reply:
We are still using JDK's httpclient for the requests (see `EnsoSecretHelper.java`).

The functionality that was missing was the `URIBuilder` that allows us to build a set of query parameters and correctly encode them. The task is not trivial as it requires escaping of various control characters depending on context (sometimes `=` sometimes not, `&` etc.) as well as escaping non-standard Unicode symbols.

We had our own simple implementation but it was missing functionalities.

Instead of trying to find the source RFC and trying to carefully implement and test that we are doing everything according to the specification, I thought it will be far simpler to use an existing solution.

If that ever turns out to be problematic, we could roll our own - but it seems like an overkill if an existing solution exists.

Note that in subsequent PRs (#8544) I plan to extend using this functionality. The `URIBuilder` allows us to easily parse the existing query and modify it in any way we need, as well as it allows us to parse and modify the URI path and all the other components. The JDK provides only very limited funcionality for that (e.g. we don't get the parsing of the query arugments that we need to be able to manipulate them).

Reply:
Btw. http://tinyurl.com/url-escaping

Reply:
There is [URLEncoder](https://docs.oracle.com/javase/8/docs/api/java/net/URLEncoder.html) which has been good enough for all purposes.

I continue to be convinced that

> adding dependency on (another) external library to `std-base`

isn't good idea.